updated list of compliance profiles

modrake opened the pull request:
This was requested by a customer; there's probably a better way to format this which I will leave to our docs team, but I wanted to get the data entry part taken care of. Thanks all!

mtyler commented:
After submitting this PR I can't help but think maybe we should just link to the RFC.

modrake committed:
updated list of compliance profiles 

This was requested by a customer; there's probably a better way to format this which I will leave to our docs team, but I wanted to get the data entry part taken care of. Thanks all!

chris-rock commented on chef_master/source/compliance.rst:
```diff
@@ -59,9 +59,59 @@ Compliance profiles are built and maintained from the Chef Compliance server web
+* CIS HP-UX 11iv3 Update 2 Benchmark Level 2
+* CIS Red Hat Enterprise Linux 6 Benchmark Level 1
+* CIS Red Hat Enterprise Linux 6 Benchmark Level 2
+* CIS Red Hat Enterprise Linux 6 Benchmark Level 1
```
CIS Red Hat Enterprise Linux **7** Benchmark Level 1

chris-rock commented on chef_master/source/compliance.rst:
```diff
@@ -59,9 +59,59 @@ Compliance profiles are built and maintained from the Chef Compliance server web
+* CIS Microsoft Windows 10 Enterprise (Release 1511) Benchmark Level 1 +BitLocker
+* CIS Microsoft Windows 10 Enterprise (Release 1511) Benchmark Level 2
+* CIS Microsoft Windows 10 Enterprise (Release 1511) Benchmark Level 2 +BitLocker
+* CIS Microsoft Windows Server 2012(non-R2) Benchmark Level 1 - Domain Controller
```
Maybe just `CIS Microsoft Windows Server 2012 Benchmark Level 1 - Domain Controller`

chris-rock reviewed (commented):
I added some minor comments, but this looks great

Commit:
Edits profile list.

Commit:
changes to Win 2012

kagarmoe commented:
![gif-keyboard-922575869955760516](https://cloud.githubusercontent.com/assets/4400151/24221079/193fe06a-0f0a-11e7-8c71-41c6c0be1a94.gif)
